autogen/oai/openai_utils.py

```diff
@@ -423,10 +425,20 @@ def config_list_from_json(
     Returns:
         List[Dict]: A list of configuration dictionaries that match the filtering criteria specified in `filter_dict`.
     """
-    json_str = os.environ.get(env_or_file)
-    if json_str:
+    env_str = os.environ.get(env_or_file)
+
+    if env_str:
+        # The environment variable exists. We should use information from it.
+        if os.path.exists(env_str):
+            # It is a file location, and we need to load the json from the file.
+            json_str = open(env_str).read()
```

Comment:
shall we use "with" for proper file operations?

Reply:
Good suggestion. Resolved.